apps/workflows/checks.py

```diff
@@ -187,3 +210,26 @@ def desc2not_equals(self, check_desc):
                     f"negative of: {doc}",
                     lambda instance: not func(instance),
                 )
+
+    def desc2in(self, statement, values):
+        if statement.count("_in") == 1:
+            attr = self.sanitize_attribute(statement[:-3])
+
+            if hasattr(self.model, attr):
+                doc = f"check that {self.model.__name__} attribute {attr} is in [{', '.join(values)}]"
+
+                def check_inclusion(instance):
+                    field = getattr(instance, attr)
+                    return (field if not callable(field) else field()) in values
+
+                return (doc, check_inclusion)
+
+    def desc2not_in(self, statement, values):
+        if statement.count("_not_in") == 1:
+            result = self.desc2in(statement, values)
```

Comment:
`_not` should be removed before calling `desc2in` - a test case should uncover that